.eslintrc.json

```diff
@@ -18,7 +18,8 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
-    "no-shadow": "off"
+    "no-shadow": "off",
+    "no-param-reassign": "off"
```

Comment:
As mentioned above, pls do not modify linters configuration files.

Reply:
@emmiiorji Thanks a lot for the feedback and tips. I will make sure the original linters are reinstated.